src/js/lightgallery.js

```diff
@@ -387,7 +388,9 @@
 
         var html;
         if (this.s.dynamic) {
-            html = this.s.dynamicEl[index].html;
+            if (this.s.dynamicEl && this.s.dynamicEl[index]) {
```

Comment:
dynamicEl should be defined when you use dynamic mode. And i think in any situation index won't go beyond the length of the array. Did you face any issue because of the old code?. If yes please explain the scenario..